pkg/workload/tpcc/tpcc.go

```diff
@@ -137,6 +141,61 @@ func (w *waitSetter) String() string {
 	}
 }
 
+var _ pgx.QueryTracer = fileLoggerQueryTracer{}
+
+type fileLoggerQueryTracer struct {
+	filePath string
+	file     *os.File
+}
+
+func newFileLoggerQueryTracer(filePath string) (*fileLoggerQueryTracer, error) {
+	tracer := &fileLoggerQueryTracer{filePath: filePath}
+	file, err := os.OpenFile(tracer.filePath, os.O_CREATE|os.O_TRUNC|os.O_RDWR, 0666)
+	if err != nil {
+		return nil, err
+	}
+	tracer.file = file
+	return tracer, nil
+}
+
+func (t *fileLoggerQueryTracer) Close() {
+	if t.file != nil {
```

Comment:
Nit (here and in other methods): IMO, we shouldn't check `file == nil`. In some cases, we're returning an error, in others we are panicking. If we just omit the check, we'll also get a panic just the same. In this `Close()` case in particular, `Close()` will just be a no-op which to me is more confusing.

More importantly, `t.file` should never be nil by construction. If somehow we have an instance floating around with a `nil` file, we want it to crash: it's not valid.

Reply:
Good point, looking back, not a fan of the inconsistency, so this is much simpler